nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -553,13 +553,18 @@ in {
           isSystemUser = true;
           home = cfg.package;
           inherit (cfg) group;
+          uid = config.ids.uids.mastodon;
```

Comment:
What was the reasoning behind switching to static uid/gids?

Reply:
Is this an variant not recommended?

Reply:
Since we only have a limited number of static uid/gids we only allocate those for services that could potentially have a lot of files that would need to be chowned in case the uid/gid ever would change.

Reply:
Ok, deleted this commit.